src/index.ts

```diff
@@ -1,10 +1,17 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License.
 
+export { InvocationContext } from './InvocationContext';

```

Comment:
Why did InvocationContext jump to the top of the list?

Reply:
No idea. We've got a few settings to automatically organize these, so I normally just let it do what it wants

For example: https://github.com/Azure/azure-functions-nodejs-library/blob/v4.x/.vscode/settings.json#L4

Reply:
Hmm, I tried to understand the rule. It seems VS Code uses ESLint's rules: https://eslint.org/docs/latest/rules/sort-imports. But reading that page, I still don't see how this happened. 😕

Reply:
I think it's because of the casing. "InvocationContext" is capital and I guess that means it prefers it before all lower case ones. Switching to use the eslint rule directly instead of through VS Code probably makes sense, but I'll look into this separate from this PR because I don't want to clutter the diff